public/_headers

```diff
@@ -0,0 +1,23 @@
+/*
+  Cache-Control: public, max-age: 0, s-maxage=900, must-revalidate
+  Content-Security-Policy: form-action https:
+  Feature-Policy: vibrate 'none'; geolocation 'none'; midi 'none'; microphone 'none'; camera 'none'; magnetometer 'none'; gyroscope 'none'; speaker 'none'; vibrate 'none'; payment 'none'
+  Referrer-Policy: no-referrer-when-downgrade
+  X-Content-Type-Options: nosniff
+  X-Frame-Options: DENY
+  X-XSS-Protection: 1; mode=block
+
+/*.js
+  Cache-Control: public, max-age: 604800, s-maxage=604800
+
+/alerts/*
+  Cache-Control: public, max-age: 1800, s-maxage=1800
+
+/alerts.json
+  Cache-Control: public, max-age: 1800, s-maxage=1800
+
+/feed.xml
```

Comment:
Why would the feed have different caching rules compared to the main website?

Reply:
You're right about `feed.xml`, still added a max-age to `alerts.json` and `ce-alert-link.js`. They both get fetched on each integrations docs page, and so it should remain cached for a longer period of time.